README.md

```diff
@@ -1,118 +1,112 @@
 # gfw_resist_tls_proxy
-# آپدیت
-رو ایرانسل پکت با موفقیت رد میشه ولی شدید کنده تحت بررسیه<br>
-این محصول نهایی نیست یه اسکریپت پایتونه جهت اثبات ادعا<br>
-بزارید به نقطه پایدار برسیم بعد برا کلاینت v2ray پیاده میشه<br>
-الان متخصص حرفه ای شبکه و برنامه نویس c لازم داریم جهت تبادل اطلاعات<br>
```

Comment:
These problems have not been solved yet, so don't delete them.
Do not delete this section.